shopify-cli.rb

```diff
@@ -95,7 +95,7 @@ def install
     ruby_libs = Dir.glob("#{prefix}/gems/*/lib")
     exe = "shopify"
     file = Pathname.new("#{brew_gem_prefix}/bin/#{exe}")
-    (bin + file.basename).open("w") do |f|
+    (bin + "#{file.basename}").open("w") do |f|
```

Comment:
What's the reason for the interpolation here?

Reply:
It doesn't add any value, just makes it easier to add the 2 in the other formula. And it doesn't change the result at all.

This is going to go away once CLI2 is only the @2 formula.